ChatQnA/tests/test_compose_on_xeon.sh

```diff
@@ -193,6 +206,16 @@ function main() {
         # validate_frontend
```

Comment:
We can use another `megaservice_stress` test to record the memory usage, which can cover the max running memory I mentioned. We can have a `mode=perfspect` for this condition.